### PR TITLE
Add abnormal length of anatomical entity pat. template

### LINKED ISSUE #855: Abnormal length of anatomical entity pattern template
1. [x] add `abnormalLengthOfAnatomicalEntity.yaml` DOS-DP pattern draft for community review
2. [x] get feedback
3. [ ] fix or improve pattern based on feedback

* Examples:
	- [HP:0040157	Abnormal intermamillary distance](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0040157)
	- [HP:0200011	Abnormal length of corpus callosum](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0200011)
	- [MP:0011999	abnormal tail length](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0011999)
	- [MP:0020999	abnormal small intestine length](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0020999)
	- [ZP:0022039	head length, abnormal](https://www.ebi.ac.uk/ols/ontologies/zp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FZP_0022039)

### PR DESCRIPTION
**Pull request by rays22**
This commit intends to add a new `deviationOfAnatomicalEntity.yaml` DOS-DP
pattern draft for community review.
If applied, this commit will fix #855.

### CODE REVIEW

File: src/patterns/dosdp-dev/abnormalLengthOfAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+equivalentTo:
+  text: "'has_part' some (
+    'length' and
+    ('characteristic_of' some %s) and
+    ('has_modifier' some 'abnormal')
+    )"
+  vars:
+    - anatomical_entity
+...
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormalLengthOfAnatomicalEntity.yaml:61]: 3 dots?

**Comment by sbello** [comment]
looks good ORCID: orcid.org/0000-0003-4606-0597

File: src/patterns/dosdp-dev/abnormalLengthOfAnatomicalEntity.yaml (hunk 1 above)
Comment (ybradford) [reply src/patterns/dosdp-dev/abnormalLengthOfAnatomicalEntity.yaml:61]: this looks good to me Orcid id 0000-0002-9900-7880

**Comment by ybradford** [comment]
this looks good to me Orcid id 0000-0002-9900-7880

File: src/patterns/dosdp-dev/abnormalLengthOfAnatomicalEntity.yaml (hunk 1 above)
Comment (rays22) [reply src/patterns/dosdp-dev/abnormalLengthOfAnatomicalEntity.yaml:61]: > 3 dots?

They are [yaml 1.2 end of document markers.](https://yaml.org/spec/1.2.0/#id2599514) My yaml linter complains if I omit them.

**Pull request merged**